core/src/main/java/org/mule/runtime/core/el/mvel/MVELExpressionLanguage.java

```diff
@@ -329,8 +330,8 @@ public static String removeExpressionMarker(String expression) {
     if (expression.startsWith(DEFAULT_EXPRESSION_PREFIX)) {
       expression = expression.substring(2, expression.length() - 1);
     }
-    if (expression.startsWith(MEL_PREFIX)) {
-      expression = expression.substring(MEL_PREFIX.length());
+    if (expression.startsWith(MEL_PREFIX + PREFIX_EXPR_SEPARATOR)) {
```

Comment:
Are you doing the same for the DW executor? Add a test case to the expression manager or somewhere where the real EL are instantiated to verify that.